js/modules/k6/browser/common/page.go

```diff
@@ -603,7 +603,11 @@ func (p *Page) getFrameElement(f *Frame) (handle *ElementHandle, _ error) {
 	for ; rootFrame.parentFrame != nil; rootFrame = rootFrame.parentFrame {
 	}
 
-	parentSession := p.getFrameSession(cdp.FrameID(rootFrame.ID()))
+	parentSession, ok := p.getFrameSession(cdp.FrameID(rootFrame.ID()))
+	if !ok {
+		return nil, errors.New("parent frame has been detached")
```

Comment:
Same question about the error text and any useful details that we could provide to the user?